tests/sentry/api/endpoints/test_organization_request_join.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import absolute_import
+
+from mock import patch
+
+from sentry.models import AuthProvider, InviteStatus, OrganizationMember
+from sentry.testutils import APITestCase
+from sentry.api.endpoints.organization_request_join import REQUEST_JOIN_EXPERIMENT
+
+
+class OrganizationRequestJoinTest(APITestCase):
+    endpoint = "sentry-api-0-organization-request-join"
+    method = "post"
+
+    def setUp(self):
+        super(OrganizationRequestJoinTest, self).setUp()
+        self.email = "test@example.com"
+        self.org = self.create_organization(owner=self.user)
+        self.owner = OrganizationMember.objects.get(user=self.user, organization=self.org)
```

Comment:
Nit: You could make this a `@fixture` so that it only makes a db call when it's actually used.